internal/pkg/cli/deploy/deploy.go

```diff
@@ -61,9 +62,13 @@ const (
 )
 
 var (
-	aliasUsedWithoutDomainFriendlyText = fmt.Sprintf("To use %s, your application must be associated with a domain: %s.\n",
+	rdwsAliasUsedWithoutDomainFriendlyText = fmt.Sprintf("To use %s, your application must be associated with a domain: %s.\n",
 		color.HighlightCode("http.alias"),
 		color.HighlightCode("copilot app init --domain example.com"))
+	ecsAliasUsedWithoutDomainFriendlyText = fmt.Sprintf("To use %s, your application must be associated with a domain: %s, or import any certificates to your environment: %s.\n",
```

Comment:
nit that I am not even sure myself:
```suggestion
	ecsAliasUsedWithoutDomainFriendlyText = fmt.Sprintf("To use %s, your application must be associated with a domain: %s, or import one or more certificates to your environment: %s.\n",
```

I was just worried about the usage of "any" would let people think that they could really import just **any** certificate and `alias` would work magically😆 I'm no English expert and I don't know if I'm fixing it right or if this even needs to be fixed.

Reply:
maybe "or import relevant certificate(s)..."?